bucardo.schema

```diff
@@ -1969,9 +1969,8 @@ FOR EACH ROW EXECUTE PROCEDURE bucardo.bucardo_delta_names_helper();
         ## Delete any tables that are no longer in the database.
         $dbh->do(q{
             DELETE FROM bucardo.bucardo_delta_targets
-             WHERE NOT EXISTS (
-                SELECT oid FROM pg_class WHERE oid = tablename
-            )});
+             WHERE NOT EXISTS (SELECT oid FROM pg_class WHERE oid = tablename)
```

Comment:
Is this just a reformatting of the existing query?  I'm not seeing a functional change here, but it is pre-coffee, so let me know if I'm missing something.